fastcrawler_ui/core/fastapi/app.py

```diff
@@ -1,10 +1,19 @@
 from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
 
 from .routers import get_routers
 
 
 def get_application(router=get_routers(), base_app=FastAPI) -> FastAPI:
     app = base_app()
+    origins = ["*"]
+    app.add_middleware(
+        CORSMiddleware,
+        allow_origins=origins,
+        allow_credentials=True,
+        allow_methods=["*"],
+        allow_headers=["*"],
+    )
```

Comment:
Not really a good approach!

Reply:
in what sense? and what's the alternative?